bayesian/skk-bayesian.el

```diff
@@ -527,5 +527,10 @@
       (write-region (point-min) (point-max) skk-bayesian-corpus-file 'append)
       (delete-region (point-min) (point-max)))))
 
+(require 'skk-bayesian)
```

Comment:
I don't understand why `(require 'skk-bayesian)` is added to skk-bayesian.el
which causes infinite loop if ruby is not found and `(skk-bayesian-init)` fails
when byte-compilation.  cf. https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=987383

Reply:
Thanks, this is my bad. Maybe I want to add `(provide 'skk-bayesian)`.